### PR TITLE
 	Bug 1108832 - Fixes for summary series support

### PR DESCRIPTION
**Pull request by wlach**
* Use a text column instead of varchar for storing series property (since the
subtest signatures can be quite long). Also stop indexing it.
* Update query for getting series signatures to not use a coallation (which
also has a size limit by default)

### CODE REVIEW

File: treeherder/model/derived/jobs.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 import MySQLdb
 import time
 import logging
+from collection import defaultdict
```
Comment (maurodoglio) [inline treeherder/model/derived/jobs.py]: as pointed out by travis, s/collection/collections

File: treeherder/model/sql/template_schema/project_jobs_1.sql.tmpl [hunk 2]
```diff
@@ -302,12 +302,11 @@ DROP TABLE IF EXISTS `series_signature`;
 CREATE TABLE `series_signature` (
   `signature` varchar(50) COLLATE utf8_bin NOT NULL,
   `property` varchar(50) COLLATE utf8_bin NOT NULL,
-  `value` varchar(150) COLLATE utf8_bin NOT NULL,
+  `value` text COLLATE utf8_bin NOT NULL,
   `active_status` enum('active','onhold','deleted') COLLATE utf8_bin DEFAULT 'active',
   KEY `idx_signature` (`signature`),
   KEY `idx_property` (`property`),
-  KEY `idx_value` (`value`),
```
Comment (maurodoglio) [inline treeherder/model/sql/template_schema/project_jobs_1.sql.tmpl]: this is not going to perform well if you need to filter based on `value`. is it the case?

Comment (wlach) [inline treeherder/model/sql/template_schema/project_jobs_1.sql.tmpl]: I can't currently think of a case where we would want to do that, but it's not inconceivable. It was a fairly arbitrary choice on my part to remove the index, maybe I should put it back.

Comment (wlach) [inline treeherder/model/sql/template_schema/project_jobs_1.sql.tmpl]: I think we agreed after chatting on irc that we should leave this index out. We can always put the index back later if we really want it. Properly indexing performance series is a complicated problem and I don't have the full solution to it yet, but it surely is much more than just putting an index on a column in this table. :)

File: treeherder/model/derived/jobs.py (hunk 1 above)
Comment (wlach) [inline treeherder/model/derived/jobs.py]: Oops, fixed.

File: treeherder/model/sql/template_schema/project_jobs_1.sql.tmpl (hunk 2 above)
Comment (maurodoglio) [inline treeherder/model/sql/template_schema/project_jobs_1.sql.tmpl]: :+1:

**Comment by maurodoglio** [comment]
@wlach did you forget to push your changes? the travis run is still failing

**Comment by wlach** [comment]
Just repushed.

**Comment by wlach** [comment]
(haven't landed yet, want to do more local testing)

**Comment by maurodoglio** [comment]
cool, I'll r+. feel free to merge it when ready

**Pull request merged**